storage/metrics.go

```diff
@@ -202,6 +202,34 @@ var (
 		Help: "Number of pending replicas in the split queue"}
 	metaSplitQueueProcessingNanos = metric.Metadata{Name: "queue.split.processingnanos",
 		Help: "Nanoseconds spent processing replicas in the split queue"}
+
+	// GCInfo cumulative totals.
+	metaGCGCKeys = metric.Metadata{Name: "queue.gc.info.gckeys",
+		Help: "Number of keys with GC'able data"}
+	metaGCIntentsConsidered = metric.Metadata{Name: "queue.gc.info.intentsconsidered",
+		Help: "Number of 'old' intents"}
+	metaGCIntentTxns = metric.Metadata{Name: "queue.gc.info.intenttxns",
+		Help: "Number of associated distinct transactions"}
+	metaGCTransactionSpanTotal = metric.Metadata{Name: "queue.gc.info.transactionspantotal",
+		Help: "Total number of entries in the transaction span"}
+	metaGCTransactionSpanGCAborted = metric.Metadata{Name: "queue.gc.info.transactionspangcaborted",
+		Help: "Number of GC'able entries corresponding to aborted txns"}
+	metaGCTransactionSpanGCCommitted = metric.Metadata{Name: "queue.gc.info.transactionspangccommitted",
+		Help: "Number of GC'able entries corresponding to commited txns"}
+	metaGCTransactionSpanGCPending = metric.Metadata{Name: "queue.gc.info.transactionspangcpending",
+		Help: "Number of GC'able entries corresponding to pending txns"}
+	metaGCAbortSpanTotal = metric.Metadata{Name: "queue.gc.info.abortspantotal",
```

Comment:
Similar comment as for the transaction span. Perhaps s/considered/scanned/ in the above comment (i.e. the `Scanned` suffix means that we had to pull it from the engine, the `Considered` suffix means that the entry was old enough so that we thought about getting rid of it).